code/game/objects/effects/decals/misc.dm

```diff
@@ -17,6 +17,7 @@
 	icon_state = "whitedecal"
 	layer = ABOVE_TURF_LAYER
 	mouse_opacity = 0
+	resistance_flags = NONE
```

Comment:
why remove the invincibility?

Reply:
This is specifically for the tile decals I created a while back, we have an issue where explosions destroy a tile leaving it plating but the decals awkwardly stay behind because invincibility.

Reply:
If there's a better way of handling this so that explosions properly delete decals on devastated tiles I'm happy to implement it.

Reply:
> If there's a better way of handling this so that explosions properly delete decals on devastated tiles I'm happy to implement it.

yea by having the decals be overlays
remove this change please

Reply:
see above response